### PR TITLE
Fix serious bug in nesterov momentum optimizer.

### LINKED ISSUE #11539: 如何禁用掉fc层的bias参数？
根据文档，fc函数的bias_attr的参数设置为None时，不会生成bias参数，但实际测试发现生成了bias参数变量，请问这个正常吗？

```
def testfc():
    place = fluid.CPUPlace()
    exe = fluid.Executor(place)
    x = fluid.layers.data(name='x', shape=[128], dtype='float32')
    fcout = fluid.layers.fc(
        name = 'testfc',
        input=x,
        size=2,
        bias_attr=None
        )
    exe.run(fluid.default_startup_program())
    for name,var in fluid.default_main_program().global_block().vars.iteritems():
        if var.persistable:
            print name
    #output 
    #testfc.w_0
    #testfc.b_0    
```

### PR DESCRIPTION
**Pull request by qingqing01**
This bug will cause the network does not converge.

Also fix https://github.com/PaddlePaddle/Paddle/issues/11539

### CODE REVIEW

**Comment by kuke** [comment]
May fix [PaddlePaddle/models/#696](https://github.com/PaddlePaddle/models/issues/696)

**Comment by qingqing01** [comment]
@kuke This bug is only in nesterov momentum, not original momentum optimizer.

File: paddle/fluid/operators/momentum_op.cc [hunk 1]
```diff
@@ -98,7 +98,7 @@ The update equations are as follows:
 $$
 velocity = mu * velocity + gradient \\
 if (use\_nesterov):   \\
-  param = param - gradient * learning\_rate + mu * velocity * learning\_rate \\
+  param = param - (gradient + mu * velocity) * learning\_rate \\
 else:   \\
   param = param - learning\_rate * velocity. \\
```
Comment (panyx0718) [inline paddle/fluid/operators/momentum_op.cc:103]: even basic momemtum seems wrong to me, mu * velocity doesn't multiply learning rate...:
```
# Momentum update
v = mu * v - learning_rate * dx # integrate velocity
x += v # integrate position
```

**Review by panyx0718 — DISMISSED** [review]

**Review by kuke — COMMENTED** [review]
@qingqing01 I see and also LGTM

**Review by panyx0718 — APPROVED** [review]
